# UbloxRaw Message

std_msgs/Header header

std_msgs/String ubloxRaw



# ubloxRaw @39 :Data;
